terraform/environment/ecs_caseworker_front.tf

```diff
@@ -23,7 +23,7 @@ resource "aws_ecs_service" "caseworker_front" {
     container_port   = 80
   }
 
-  depends_on = [aws_db_instance.applications, aws_lb.caseworker_front, aws_iam_role.caseworker_front_task_role, aws_iam_role.execution_role]
+  depends_on = [aws_rds_cluster.applications, aws_lb.caseworker_front, aws_iam_role.caseworker_front_task_role, aws_iam_role.execution_role]
```

Comment:
I think this is a mistake I already had here, but I don't think caseworker_front is dependant on the database. It might be dependant on caseworker_api. Removing the dependency will speed up build time slightly.